pkgs/misc/screensavers/pipes-rs/default.nix

```diff
@@ -0,0 +1,26 @@
+{ rustPlatform, fetchFromGitHub, lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pipes-rs";
+  version = "1.4.4";
+
+  src = fetchFromGitHub {
+    owner = "lhvy";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "drqoKkju1EkcWGNnliEah37wVhtU2ddJSOZ5MnCNbuo=";
+  };
+
+  cargoLock = { lockFile = ./Cargo.lock; };
```

Comment:
The reason behind this action was the different sha256 produced by the local package build compared to the build performed by Ofborg. Thus we I thought it would've been easier to use a cargo.lock file instead. 

Maybe that was not the wisest choice and that the sha256 was different because @VergeDX built it in his MacOS system, I don't know. Do enlighten us if you do know something that we are not aware of! :)